install/jsonp/docs/index.html

```diff
@@ -45,7 +45,7 @@ <h2>Guide to Jakarta JSON Processing TCK 2.0 Documentation</h2>
 <ul>
   <li>The <em><a href="JSONP2.0-ReleaseNotes.html">Jakarta JSON Processing TCK Release Notes</a></em> provides the information on the test harness, the test suite and the platforms tested.
     <ul>
-      <li>If needed, updated  Release Notes are available at the Jakarta EE Specification web site for Jakarta JSON Processing: <a href="https://jakarta-ee/specifications/json-processing/2.0/">https://jakarta-ee/specifications/json-processing/2.0/</a></li>
+      <li>If needed, updated  Release Notes are available at the Jakarta EE Specification web site for Jakarta JSON Processing: <a href="https://jakarta.ee/specifications/jsonp/2.0/">https://jakarta.ee/specifications/jsonp/2.0/</a></li>
     </ul>
   </li>
   <li>The <em>Jakarta JSON Processing Technology Compatibility Kit, Version 2.0 User's Guide </em> provides the information that you need to install, set up, and  run  the Jakarta JSON Processing TCK, Version 2.0.  In addition, the guide provides the rules  you must comply with to pass the Jakarta JSON Processing TCK.</li>
```

Comment:
Can you please also address this here as mentioned in https://github.com/eclipse-ee4j/jakartaee-tck/issues/433 
"Jakarta JSON Processing Technology Compatibility Kit, Version 2.0 User's Guide in the docs/index.html should be clickable link to actual user guide contained in the tck zip"

The link to the first page for html usersguide will be at html-usersguide/toc.html in the tck bundle (eg. https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/jakarta-persistence-tck-3.0.0.zip)